iceoryx_posh/include/iceoryx_posh/popo/smart_chunk.hpp

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
```

Comment:
If you want, you can also add your copyright here

Reply:
Done